portfolio/src/main/webapp/script.js

```diff
@@ -88,30 +88,52 @@ function getComments() {
 getComments();
 
 /**
- * Checks login status of user, links to login/logout page, hides comment form
- * if user is logged out.
+ * Fetches login status data from /login and updates comment-form div in DOM
+ *     depending on login status. Depending on status, DOM is updated to hide
+ *     the form and prompt the user to login or out.
  */
-function checkLoginStatus() {
+function updateCommentForm() {
   fetch('/login')
   .then(response => response.json())
   .then((status) => {
-    const baseUrl = window.location.origin;
-    let url = new URL(status[1], baseUrl);
-
     let commentForm = document.getElementById('comment-form');
-    let para = document.createElement("p");
     
-    if (status[0] === "True") {
-      para.innerHTML = "<p>Logout <a href=\"" + url + "\">here</a>.</p>";
+    if (status['isLoggedIn'] === true) {
```

Comment:
Is it required to explicitly check if its true? or can it just be `if (status['isLoggedIn'])`?